swan-lake/learn-the-platform/ballerina-tooling/cli-documentation/graphql.md

```diff
@@ -24,6 +25,22 @@ The command-line arguments below can be used with the command for each particula
 | `-i, --input`  | The `input` parameter specifies the path of the GraphQL config file (e.g., `graphql.config.yaml`) configured with GraphQL schemas specified by the Schema Definition Language and GraphQL documents. This parameter is mandatory.                             |
 | `-o, --output` | The `output` parameter specifies the path of the output location of the generated files. This parameter is optional. If this parameter is not specified, the Ballerina files will be generated at the same location in which the GraphQL command is executed. |
 
+### Service generation 
+```
+$ bal graphql [-i | --input] <graphql-schema-file-path>
+            [-o | --output] <output-location>
+            [-m | --mode] <operation-mode>
+            [-r | --use-records-for-objects]
+```
+The command-line arguments below can be used with the command for each particular purpose as described below.
+
+| Argument       | Description                                                                                                                                                                                                                                                   |
```

Comment:
```suggestion
| Argument       | Description                                                                                                                                                                                                                                                   |
```
Shall we have a new column to mention whether the parameter is mandatory or optional?

Reply:
There are two other tables with the same columns. It's better to keep this as it is, to be consistent with them.